src/app/globals.css

```diff
@@ -45,6 +45,16 @@
     --border: 240 6% 90%;
     --input: var(--border);
     --ring: 224 71.4% 4.1%;
+
+    --toast-success: 137 64% 98%;
+    --toast-error: 0 78% 98%;
+    --toast-warning: 40 79% 96%;
+    --toast-info: 218 80% 98%;
+
+    --sm: 640px;
+    --md: 768px;
+    --lg: 1024px;
+    --xl: 1280px;
```

Comment:
The dark theme shouldn't include (override) these values

Reply:
Ah, I wasn't sure if it overrides it, and then forgot to test it, thanks, removed in https://github.com/guildxyz/guild.xyz/pull/1333/commits/f9f02fad29940fb4d782cc0068eadf7fa13cce10